packages/reports/src/Constants/data-table-constants.js

```diff
@@ -439,7 +439,7 @@ export const getAccumulatorOpenPositionsColumnsTemplate = ({ currency, onClickSe
         },
     },
     {
-        title: localize('Accumulate'),
+        title: localize('Accumulator'),
```

Comment:
@henry-deriv why are we changing this one?😊
it's still 'Accumulate' in the design:
<img width="1249" alt="Screenshot 2023-02-14 at 9 02 09 PM" src="https://user-images.githubusercontent.com/103177211/218820173-552c2d8c-d637-43c6-ba78-104918622e05.png">

Reply:
Huh. Seems its different on my design file. 

https://www.figma.com/file/BXihIz3cSb2SC9U7kQ64pX/DTrader?node-id=6280%3A513997&t=7jdBxXDAamiUdxzE-0